gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/MysqlMultiActiveLeaseArbiter.java

```diff
@@ -221,6 +237,31 @@ private void initializeConstantsTable() throws IOException {
     }, true);
   }
 
+  /**
+   * Periodically deletes all rows in the table with event_timestamp older than the retention period defined by config.
+   */
+  private void runRetentionOnArbitrationTable() {
```

Comment:
Small nit but can we add the code for starting a thread within this method? Due to how it's implemented as an infinite loop we would generally always want this function to be run asynchronously